k8s/eks-setup.md

```diff
@@ -0,0 +1,230 @@
+
+https://docs.aws.amazon.com/AmazonECR/latest/userguide/getting-started-cli.html
+
+Create an ECR repo for zqd:
+
+```
+aws ecr create-repository \
+    --image-scanning-configuration scanOnPush=true \
+    --repository-name zqd
+```
+
+When the repo is created at the command line it returns JSON that looks like:
+```
+"repositoryUri": "123456789012.dkr.ecr.us-east-2.amazonaws.com/gateway"
+```
+Sustitute this URI into the tag, login and push steps. To push the zqd image created with the local build:
+```
+make docker
+docker tag zqd 123456789012.dkr.ecr.us-east-2.amazonaws.com/zqd
+aws ecr get-login-password --region us-east-2 | docker login \
+  --username AWS --password-stdin 123456789012.dkr.ecr.us-east-2.amazonaws.com
+docker push 123456789012.dkr.ecr.us-east-2.amazonaws.com/zqd
+```
+## Use zar import to get some data into an S3 bucket in the region
+We will use zar to create an S3 bucket in this region with sample data. 
+
+If you do not already have a bucket you want to use in the same region as your EKS cluster, create it with `aws s3 mb`, e.g.
+```
+aws s3 mb s3://zqd-demo-1
+```
+Now use zar to create an object in S3. 
+
+For the following zar import, change the directory name to match your s3 bucket.
+```
+zar import -R s3://zqd-demo-1/mark/sample-http-zng zq-sample-data/zng/http.zng.gz
+```
+This creates zng files in an s3 directory called `sample-http-zng` that we will use from zapi after zqd has been deployed. To check what zar created:
+```
+aws s3 ls zqd-demo-1/mark --recursive
+```
+
+## Deploy the zqd service with Helm
+Here is an example Helm install. This assumes you created an S3 resident space as in the local install above. This is similar to the local Helm deploy. It uses the same charts, but command line parameters overide the Values.yaml to provide specific configuration for EKS.
+
+Substitute the image.repository you created above. Note that unlike the local deployment, we do not use K8s secrets for AWS credentials because the `cluster.yaml` above specified IAM policies for S3 access.
+
+```
+helm install zqd-test-1 charts/zqd \
+  --set AWSRegion="us-east-2" \
+  --set datauri="s3://zqd-demo/mark/zqd-meta" \
+  --set image.repository="123456789012.dkr.ecr.us-east-2.amazonaws.com/" \
+  --set useCredSecret=false
+```
+
+## Exposing the zqd endpoint
+
+To access the running zqd instance, use kubectl port-forward:
+```
+kubectl port-forward svc/zqd-test-1 9867:9867 &
+```
+There is a script `k8s/zqd-port.sh` that does this after removing any existing port-forwards.
+
+## Using a K8s hosted zqd to search large logs in the cloud
+This decribes a set of steps to use `zar import` to process large log files that can then be searched with the zqd you just deployed. In order to do this, you will need an EC2 instance in the same region as your EKS cluster. Follow the steps below to set this up.
+
+## Setting up an EC2 instance to run zar import
```

Comment:
As commented above you can just post data into an archive space using `zapi post`. As long as these file(s) specified are in s3 buckets this will work fine. I think this section can go away.

Reply:
I'm pulling this section out and moving it to "trouble-shooting.md"